src/main/java/org/broadinstitute/hellbender/tools/walkers/variantutils/CopyFilters.java

```diff
@@ -0,0 +1,4 @@
+package org.broadinstitute.hellbender.tools.walkers.variantutils;
```

Comment:
@lbergelson did you mean to include this file?

Reply:
Oh man, I definitely did not.

Reply:
I've just been `commit -a`ing all sorts of things this week...